ansible/iam.yml

```
@@ -1,8 +1,42 @@
-- hosts: basic_users
+- hosts: freeipa_client
+  tags:
+    - freeipa
+    - freeipa_server # as this is only relevant if using freeipa_server
+    - freeipa_host
+  gather_facts: no
   become: yes
+  tasks:
+    - name: Add freeipa hosts
+      import_role:
+        name: freeipa
+        tasks_from: addhost.yml
+      when: groups['freeipa_server'] | length > 0
+
+- hosts: freeipa_client
   tags:
-    - basic_users
+    - freeipa
+    - freeipa_client
   gather_facts: yes
+  become: yes
+  tasks:
+    - name: Install freeipa client
+      import_role:
+        name: freeipa
+        tasks_from: client-install.yml
+    - name: Enrole freeipa clients
+      import_role:
+        name: freeipa
+        tasks_from: enrole.yml
```

```suggestion
        tasks_from: enrol.yml
```

Needs corresponding change of filename